ricecooker/classes/nodes.py

```diff
@@ -212,6 +216,50 @@ def get_json_tree(self):
 
         return tree
 
+
+    def save_channel_children_to_csv(self, metadata_csv, topic_structure = list()):
```

Comment:
Now that the reference error is fixed, you can just rename `topic_structure` to `structure_list`, and remove all references to `topic_structure`.